### PR TITLE
Docker fixup

### PR DESCRIPTION
**Pull request by mvonbun**
Docker messed up with paths in synctex. Fixed it. Plus added some magic to the Makefile and latexmkrc.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+$(warning *** Could not determine main LaTeX file, trying to use $(DISS))
+$(warning *** - either delete all files containing '\documentclass')
+$(warning *** - or specify your main file in the Makefile)
+endif
+
 TEX      = latexmk
 TEXFLAGS = -recorder -pdf
 TEXCLEAN = -bibtex -c
 
-DOCKER   = docker run -it --rm -v $(shell pwd):/diss \
```
Comment (andre-richter) [inline Makefile:7]: Das Dockerfile hat /diss hardcoded.

**Review by andre-richter — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (mvonbun) [reply Makefile:7]: I know, that's why I added the -w $(pwd) switch to the docker run command, which acts like a WORKDIR HOSTPWD in the Dockerfile when building the container.

The main problem is that files like the synctex.gz uses the paths of the docker container (e.g. "Input:1:/diss/./dissertation.tex"), which might render the synctex file useless for the host system.

Comment (andre-richter) [reply Makefile:7]: Okay sorry, my question was more like do we still need the workdir assignment in the Dockerfile if it's added here explicitly? Only on mobile right so can't check. If not we it's obsolete there  and can be deleted, right?

Comment (mvonbun) [reply Makefile:7]: Yep, imho we don't need the Workdir assignment in Dockerfile. I removed it in the latest commit, but haven't tested it.

**Comment by andre-richter** [comment]
Haste inzwischen Mal getestet ohne das diss im Dockerfile? Dann msch ich meinen Change request auf grün.

**Comment by mvonbun** [comment]
Jop, is getestet und läuft.

**Review by andre-richter — APPROVED** [review]

**Pull request merged**